nntrainer/include/neuralnet.h

```diff
@@ -123,23 +123,23 @@ class NeuralNetwork {
    * @param[in] input Input Tensor X
    * @retval    Output Tensor Y
    */
-  Tensor forwarding(Tensor input);
+  Tensor forwarding(Tensor input, int &status);
 
   /**
    * @brief     forward propagation
    * @param[in] input Input Tensor X
    * @param[in] label Input Tensor Y2
    * @retval    Output Tensor Y
    */
-  Tensor forwarding(Tensor input, Tensor output);
+  Tensor forwarding(Tensor input, Tensor output, int &status);
```

Comment:
Could you add more docs for `Tensor output`? 
I think this is output label tensor while `@retval` is output score tensor but not confident.

Reply:
No. We couldn't. This function is used for the run forwarding only.